docs/receipt-verifier.md

```diff
@@ -118,7 +118,13 @@ Add the following client-side code to your website to listen for a `monetization
 </head>
 ```
 
-Your backend can send the receipt to the receipt verifier service’s `/verify` endpoint. Here’s an example for an Express.js server:
+Your backend can send the receipt to the receipt verifier service’s `/verify` endpoint.
```

Comment:
`/verify` endpoint could maybe link to https://github.com/coilhq/receipt-verifier#post-verify